src/stim/cmd/command_diagram.pybind.cc

```diff
@@ -55,17 +72,23 @@ void stim_pybind::pybind_diagram_methods(pybind11::module &m, pybind11::class_<D
             out << R"HTML(<img style="max-width: 100%; max-height: 100%" src="data:image/svg+xml;base64,)HTML";
             write_data_as_base64_to(self.content.data(), self.content.size(), out);
             out << R"HTML("/></div>)HTML";
-            return pybind11::cast(out.str());
+            output = out.str();
         }
         if (self.type == DIAGRAM_TYPE_GLTF) {
             std::stringstream out;
             write_html_viewer_for_gltf_data(self.content, out);
-            return pybind11::cast(out.str());
+            output = out.str();
         }
         if (self.type == DIAGRAM_TYPE_HTML) {
-            return pybind11::cast(self.content);
+            output = self.content;
+        }
+	if (output == "None"){
+	    return pybind11::none();
+        } else {
+            std::string prefix = "<iframe width=950 height=650 srcdoc=\"";
+            std::string postfix = "\"></iframe>";
```

Comment:
If you refactor the `escape` method into a `wrap html in iframe` method, you can avoid the need for the `"None"` sygil by just wrapping inplace. I'd actually prefer that, since it's not obvious to me that we unconditionally want to wrap all html.